packages/meetup-shared/src/meetupIssueCommentStatus.ts

```diff
@@ -0,0 +1,71 @@
+import { ValidationErrors } from './formatValidationErrors';
+
+type Step =
+  | {
+      status: 'idle' | 'loading' | 'success';
+      error?: never;
+    }
+  | {
+      status: 'error';
+      errors: ValidationErrors | object;
+    };
+
+export function getMeetupIssueCommentStatus(
+  steps: readonly [Step, Step, Step],
```

Comment:
Jätetään alla olevat indeksillä lukemiset C devaajille ja otetaan destructurointi tänne.

```suggestion
  [firstStep, secondStep, thirdStep]: readonly [Step, Step, Step],
```